app/static/src/scss/includes/_browse.scss

```diff
@@ -322,6 +323,7 @@
   .govuk-button__search-button {
     width: 50%;
     margin-bottom: 1rem;
+    margin-left: 4px;
```

Comment:
Do we need this margin-left

Reply:
Yup, it was to increase the space between the box and the button as per terrys design

Reply:
does it work with mobile view ?

Reply:
No, I fixed the desktop view, if ticket covers the mobile styling as well will move to WIP

Reply:
discussion during the sprint review, Colin mentioned covering the mobile view in his ticket, I think this ticket is limited to the desktop view only.